compilers/src/make.js

```diff
@@ -2,8 +2,8 @@ const fs = require('fs')
 
 const domains = fs.readFileSync('../../domains', 'utf8').split('\n')
 
-fs.appendFile('hostfile', '##COINHIVE DOMAINS\n')
+fs.appendFile('../../hostfile', '##COINHIVE DOMAINS\n', ()=>{})
 domains.forEach((domain) => {
-  fs.appendFile('../../hostfile', '0.0.0.0 ' + domain + '\n')
+  fs.appendFile('../../hostfile', '0.0.0.0 ' + domain + '\n', ()=>{})
 })
-fs.appendFile('hostfile', '##END COINHIVE\n')
+fs.appendFile(../../'hostfile', '##END COINHIVE\n', ()=>{})
```

Comment:
This is written outside of a string. Have you even tested this, because the program doesn't run.

Reply:
This is showing inconsistencies on my machine. One or two domains being appended to the hostfile after the footer.

Reply:
Well, it was a quick edit. Changing it atm.